extras/bench/test_Simple.py

```diff
@@ -3,9 +3,10 @@
 Simple FunkLoad test
 '''
 import unittest
-from random import random
+from random import random # noqa
```

Comment:
What is noqa?

Reply:
#noqa is a keyword for flake8 functionality. It makes flake8 skip the line preventing errors where ther should be none.